src/services/authenticator.js

```diff
@@ -25,26 +27,27 @@ function Authenticator() {
     }
   };
 
+  this.pathToForestrc = `${os.homedir()}/.forestrc`;
+
+  this.saveToken = (token) => fs.writeFileSync(this.pathToForestrc, token);
+
   this.verify = (token) => {
-    const decodedToken = jwtDecode(token);
+    if (!token) return null;
+    let decodedToken;
+    try {
+      decodedToken = jwtDecode(token);
+    } catch (error) {
+      return null;
+    }
     const nowInSeconds = Date.now().valueOf() / 1000;
     if (decodedToken.exp && nowInSeconds < decodedToken.exp) {
       return token;
     }
     return null;
   };
 
-  this.login = (config) =>
-    agent
-      .post(`${config.serverHost}/api/sessions`, {
-        email: config.email,
-        password: config.password,
-      })
-      .then((response) => response.body)
-      .then((auth) => {
-        config.authToken = auth.token;
-        return fs.writeFileSync(`${os.homedir()}/.forestrc`, auth.token);
-      });
+  this.validateToken = (token) => !!this.verify(token) || 'Invalid token. Please enter your'
+    + ' authentication token.';
```

Comment:
This two lines could be refactored to be more readable (just a suggestion though)